app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/slack-channels-guide.mdx

```diff
@@ -0,0 +1,383 @@
+---
+meta:
+  title: Virtual Coffee Slack Channels Guide
+  description: List of channels in our community's Slack.
+hero:
+  Hero: UndrawQuickChat
+tags: [memberresources]
+order: 3
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+We provide you here with the complete list of our Slack channels — in alphabetical order — and the purpose of each channel! Don't forget to check the bookmarks and pinned messages on the top of each channel, if any, to find helpful links and resources.
+
+If you need to familiarize yourself with Slack, visit our <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee#tips-around-slack">Tips Around Slack</Link> section.
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## A
+
+### #accessibility
+
+This channel is all about questions, comments, tips, and answers around accessibility.
```

Comment:
This channel is all about questions, comments, tips, and resources related to accessibility.